src/containers/main/UpgradeContainer/index.tsx

```diff
@@ -235,6 +235,33 @@ export const UpgradeContainer: FC<IProps> = ({ address, balance }) => {
 		setHideLowBalance(!hideLowBalance);
 	};
 
+	const addToMetamask = async (address: string, symbol: string, token: any) => {
```

Comment:
Please either create an Interface for the token param or use predefined types

Reply:
done